src/Sylius/Behat/Context/Domain/ProductContext.php

```diff
@@ -163,4 +163,12 @@ public function thereAreNoVariants(ProductInterface $product)
     {
         expect($this->productVariantRepository->findBy(['object' => $product]))->toBe([]);
     }
+
+    /**
+     * @Given I am logged in as an administrator
+     */
+    public function iAmLoggedInAsAnAdministrator()
```

Comment:
IMO we should extract this method to e.g. `.../Domain/SecurityContext` to not duplicate it on every domain context that needs it (PromotionContext, OrderContext, ProductContext etc.).